### ISSUE TITLE
Mesh-to-LAN traffic restriction not working

### ISSUE THREAD

**Issue by amuuza** [sbmesh/sbmesh-profiles#1]
Hello!!

We are amuza (for Github I am amuuza) and @2Vera, two people building a community network with LibreMesh. We found by chance your work on sbmesh and found it very interesting. We have been looking for a solution like yours for a long time, but still do not get it. We mean what you explained in method one at
https://www.sbmesh.net/join.html#diy
when you say "Thanks to our firewall configuration, users on the mesh will not be able to access your home LAN."

That is exactly what we are looking for. We asked for help to the LibreMesh community but we don't have a working solution yet, mainly because we are not coders and we need clear instructions on what to configure.

Yesterday we downloaded your firmware, managed to flash a valid NSM5 and tested it. Unfortunately, we saw users in the mesh could access the home LAN.

Why did it not behave as you described at https://www.sbmesh.net/join.html#diy ?

If we knew it works for sbmesh, we could copy/adapt that configuration into our LibreMesh network profile.

Did it ever work for you?

What should we do?

It would be awesome if you could contribute your knowledge and experience in order to solve the issue I opened at the network-profiles repository of the LibreMesh project at Github:
https://github.com/libremesh/network-profiles/issues/24

Thank you!!

**Comment by raylas** [comment]
Hi @amuuza !

Thanks for opening this issue.

Our website documentation dates from when we were using qMp for our firmware (vs. LibreMesh). I saw that you asked this same question on the LibreMesh mailing list. I think their responses answers your question well.

However, I will be updating this repo—[or the one here](https://github.com/libremesh/network-profiles) once we merge—with our iptables configuration and will close this issue then.

**Comment by amuuza** [comment]
Hi @raylas !

Thanks for answering. I just adapted a couple of lines of your configuration and so far that seems to be enough to do the job!! So thanks!!
They are:
ptables -I FORWARD -d 192.168.0.0/16 -j DROP
iptables -I FORWARD -d 172.16.0.0/12 -j DROP

I am learning a bit of iptables in order to understand the rest of your rules but, maybe you can tell me a bit of what they are for?
What is line 09 of https://github.com/sbmesh/sbmesh-profiles/blob/master/ubnt-nano-m-xw/default/etc/uci-defaults/zfirewall doing? What is it for?

And, did you remove your firewall configuration? Why? Wasn't it working?
Or maybe you just prefer everything completely open...

Cheers!

**Comment by raylas** [comment]
@amuuza,

Those lines exist but are commented out in our `default/etc/uci-defaults/zfirewall` shell script. I just needed to uncomment them!

The rest of the entries are a holdover from using not only qMp, but copying the configuration from NYC Mesh's repo. They were extra iptables entries for tinc-based VPN usage and mesh-wide management over SSH.

As I saw you ask [here](https://github.com/libremesh/network-profiles/issues/24), there is some good discussion on the current build of LiMe and why things appear to be "completely open."

Thanks for the inquiry!
